de.dashup.application/src/main/webapp/components/dashup-grid-layout.js

```diff
@@ -0,0 +1,56 @@
+import {LitElement, html, css} from "https://unpkg.com/lit-element@2.1.0/lit-element.js?module";
+import {DashupComponent} from "./dashup-component.js";
+export class DashupGridLayout extends DashupComponent{
+
+    render() {
+        return html`
+            <div class="container">
+                ${this.elements.map((row) => html`
+                    <div class="row">
+                        ${row.map((element) => {
+            let size = element.layout.size;
+            let offset = element.layout.offset;
+            return html`
+                                <div class="col s${size} m${size} l${size} xl${size} 
+                                            offset-s${offset} offset-m${offset} offset-l${offset} offset-xl${offset}">
```

Comment:
I think this mechanism is kinda complex when it comes to use `DashupGridLayout`. We should simplify that, for instance leaving the attributes `row` and `offset` out.

Reply:
A row is definitely a must in order to use a grid layout in another grid layout. This way you can define any custom layout you want. If the user does not need an offset, then he could simply leave it away. I could implement a boolean binding if the offset value has been set.

Reply:
I made it optionally to set a size and offset for components. The default size is set to 4, whereas default offset is set to 0.